uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/leave-request.controller.js

```diff
@@ -5,23 +5,48 @@ define([
 ], function (controllers) {
```

Comment:
I wonder if we can give both the files and the controllers more meaningful names. `leave-request.controller` and "LeaveRequestCtrl" (and their "sick" and "toil" variations) are so generic, they could be really about anything, you have no context.

We are talking about sub controllers of the details tab of a request modal, so maybe something like:
```
request-modal-details-leave.controller.js -> "RequestModalDetailsLeaveController"
request-modal-details-sick.controller.js -> "RequestModalDetailsSickController"
request-modal-details-toil.controller.js -> "RequestModalDetailsToilController"
```
?

Reply:
Done.